src/status_im/config.cljs

```diff
@@ -24,7 +24,7 @@
 (goog-define ALCHEMY_OPTIMISM_MAINNET_TOKEN "")
 (goog-define ALCHEMY_OPTIMISM_GOERLI_TOKEN "")
 (goog-define ALCHEMY_OPTIMISM_SEPOLIA_TOKEN "")
-(goog-define WALLET_CONNECT_PROJECT_ID "")
```

Comment:
could you rebase and push without the project id? I'll add a task now to add it to the secrets so we don't expose it. It's kinda unclear what to do with it, but for the extra safety we should not push it for now